components/upload/UploadList.tsx

```diff
@@ -121,14 +121,14 @@ export default class UploadList extends React.Component<UploadListProps, any> {
           {file.name}
         </a>
       ) : (
-        <span
-          className={`${prefixCls}-list-item-name`}
-          onClick={e => this.handlePreview(file, e)}
-          title={file.name}
-        >
-          {file.name}
-        </span>
-      );
+          <span
+            className={`${prefixCls}-list-item-name`}
+            onClick={e => this.handlePreview(file, e)}
+            title={file.name}
+          >
+            {file.name}
+          </span>
+        );
```

Comment:
why change indent?

Reply:
Sorry,it was completed by my editor automatically.And I reckoned that it was in tslint.config by mistake.
I will push a new one without changing indent.

Reply:
You can continue to push here instead of creating a new one.